src/index.js

```diff
@@ -31,6 +32,21 @@ const store = configureStore(reduxState)
 
 const history = syncHistoryWithStore(browserHistory, store)
 
+// Check if token existed in localStorage and expired
+// token can be stored in localStorage in two scenario
+// 1. oAuth
+// 2. TWReporter account sign in
+const { auth } = store.getState()
+if(auth.authenticated && auth.authInfo && (auth.authType=== 'facebook' || auth.authType==='google')) {
+  setupTokenInLocalStorage(auth.authInfo)
+  store.dispatch(deletAuthInfoAction())
+}
+
+// 7 = 7 days
+store.dispatch(authUserByTokenAction(7, auth.authType))
+  .then(() => {})
+  .catch(() => {})
```

Comment:
is this not complete yet?

Reply:
I need the promise handler to prevent errors. But, I think that the front end app no need to do anything when the auth info is not existed or is expired in localStorage.

Reply:
Since you don't do any business logic after the action, why do you need to make it as a `Promise`?